### PR TITLE
Cherry pick #930 to release-1.0

### PR DESCRIPTION
**Pull request by aylei**
cherry-pick #930 to release-1.0

### CODE REVIEW

**Comment by aylei** [comment]
@weekface e2e passed, PTAL

File: tests/cmd/stability/main.go [hunk 1]
```diff
@@ -191,12 +191,16 @@ func run() {
 			cluster.TiKVPreStartScript = strconv.Quote("")
 			cluster.TiDBPreStartScript = strconv.Quote("")
```
Comment (weekface) [inline tests/cmd/stability/main.go]: do we need to add "# noop" to `TiDBPreStartScript` in this PR?

Comment (aylei) [reply tests/cmd/stability/main.go]: I prefer adding it

**Comment by aylei** [comment]
closed in favor of #1029 @weekface

**Pull request closed without merging**